docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```petab1
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0)
+nothing # hide
+```
+
+### Fitting parameters
+We are now able to fit our model to the data. First, we create a `PEtabODEProblem`. Here, we use `petab_model` as the only input, but it is also possible to set various [numeric solver and automatic differentiation option](@ref petab_simulation_conditions) (such as method or tolerance).
+```petab1
+petab_problem = PEtabODEProblem(petab_model)
+nothing # hide
+```
+To fit a parameter set we use the `calibrate_model` function. In addition to our `PEtabODEProblem`, we must also provide an initial guess (which can be generated with the `generate_startguesses` function) and an optimisation algorithm (which needs to be imported specifically). PEtab.jl supports various [optimisation methods and options](@ref petab_optimisation_optimisers). 
+```petab1
+using Optim
+p0 = generate_startguesses(petab_problem, 1)
+res = calibrate_model(petab_problem, p0, IPNewton())
+```
+
+We can now simulate our model for the fitted parameter set, and compare the result to the measurements and true solution.
+```petab1
+oprob = ODEProblem(rn, u0, (0.0, 10.0), get_ps(res, petab_problem))
+
+sol = solve(oprob, Tsit5(); saveat=0.1)
+plot!(sol; idxs=4, label="Fitted solution")
+```
+Here we use the `get_ps` function to retrieve a full parameter set using the optimal parameters (alternatively, the `ODEProblem` or fitted simulation can be retrieved directly using the `get_odeproblem` or `get_odesol` [functions](https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.get_odeproblem), respectively). The calibration result can also be found in `res.xmin`, however, note that PEtab automatically ([unless a linear scale is selected](petab_parameters_scales)) converts parameters to logarithmic scale, so typically `10 .^res.xmin` are the values of interest. If you investigate the result from this example you might note that the found parameter set (while it fits the data well) does not correspond to the true parameter set. This phenomenon is related to *identifiability*, and is very important for parameter fitting.
+
+### Final notes
+PEtab.jl also supports [multistart optimisation](@ref petab_multistart_optimisation), [automatic equilibration before simulations](https://sebapersson.github.io/PEtab.jl/stable/Brannmark/), and [events](@ref petab_events). Various [plot recipes](@ref petab_plotting) exist for investigating the optimisation process. Please read the [PETab.jl documentation](https://sebapersson.github.io/PEtab.jl/stable/) for a more complete description of the package's features. Below follows additional details of various options and features (generally, PEtab is able to find good default values for most options not specified).
+
+## [Additional features: Observables](@id petab_observables)
+
+### [Defining non-trivial observables](@id petab_observables_observables)
```

Comment:
I think it would also be nice to mention scale and offset parameter here? as they are quite common (as is done here https://sebapersson.github.io/PEtab.jl/stable/Julia_obs_noise/)

Reply:
added